layers/+completion/ivy/config.el

```diff
@@ -47,3 +47,6 @@ than this amount.")
         for desc = (nth 2 j)
         collect `(,key (lambda (x) (spacemacs//counsel-with-git-grep (quote ,func) x)) ,desc))
   "Default ivy actions to be used with git-grep output.")
+
+(defvar spacemacs--ivy-rich nil
+  "Whether include ivy-rich or not.")
```

Comment:
Maybe "If non-nil, enable `ivy-rich'."

Reply:
Updated and thanks. I'll catch up on naming and doc conventions.